pkg/controller/tailoredprofile/tailoredprofile_controller.go

```diff
@@ -348,6 +448,10 @@ func newTailoredProfileCM(tp *compliancev1alpha1.TailoredProfile) *corev1.Config
 		"tailored-profile": tp.Name,
 	}
 	return &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
```

Comment:
Why is it needed to explicitly specify the `TypeMeta`?

Reply:
Ah, I guess in order to convert the CM to Unstructured?

Reply:
Exactly